packages/lesmis-server/src/routes/UsersRoute.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import { Request, Response, NextFunction } from 'express';
+import uniqby from 'lodash.uniqby';
+import { Route } from '@tupaia/server-boilerplate';
+import { MeditrakConnection } from '../connections';
+
+const FIELDS: Record<string, string> = {
+  user_id: 'id',
+  'user.creation_date': 'createdAt',
+  'user.first_name': 'firstName',
+  'user.last_name': 'lastName',
+  'user.email': 'email',
+  'user.mobile_number': 'mobileNumber',
+  'user.employer': 'employer',
+  'user.position': 'position',
+  'entity.name': 'entityName',
+  'entity.code': 'entityCode',
+  'permission_group.name': 'permissionGroupName',
+  id: 'userEntityPermissionId',
+};
+
+const PERMISSION_GROUPS = ['Laos Schools Admin', 'LESMIS Public', 'Laos Schools Super User'];
+
+// Remove duplicat records and convert user keys to nice camelCase keys for the front end
+const getProcessedUserData = (userData: { user_id: number }[]) =>
+  uniqby(userData, 'user_id').map((user: Record<string, string[]>) => {
+    const newUser: Record<string, string[]> = {};
+    const keys = Object.keys(user);
+    keys.forEach(key => {
+      const newKey = FIELDS[key];
+      newUser[newKey] = user[key];
+    });
+    return newUser;
+  });
+
+export class UsersRoute extends Route {
+  private readonly meditrakConnection: MeditrakConnection;
+
+  constructor(req: Request, res: Response, next: NextFunction) {
+    super(req, res, next);
+
+    this.meditrakConnection = new MeditrakConnection(req.session);
+  }
+
+  async buildResponse() {
+    const userData = await this.meditrakConnection.getUserEntityPermissions({
+      pageSize: '10000',
```

Comment:
```suggestion
      pageSize: '10000', // retrieve all entries, as we don't handle pagination
```
10000 might be too small for LESMIS in the end, considering there are more schools than that

@enunan how many users are we expecting?